MapboxDirections/Match/MBMatchOptions.swift

```diff
@@ -0,0 +1,82 @@
+import Foundation
+
+@objc(MBMatchOptions)
+open class MatchOptions: RouteOptions {
```

Comment:
MatchOptions sounds even more like a method name than RouteOptions. Should we call it MatchingOptions, or is it more important to keep the two class names parallel?

Reply:
MatchingOptions workings